### DISCUSSION TITLE
Will scores from lazer.ppy.sh get migrated to the stable database or are they gone for good?

### DISCUSSION THREAD

**Discussion by SamedhG** [ppy/osu#26989] [Q&A]
Just curious about this, not a huge concern but it would be nice since I essentially only played on lazer all of 2023

**Comment by bdach** [comment]
Still up in the air. As soon as a conclusive decision is reached we'll let know.

Playlist scores will likely be brought across. The rest is a bit difficult to determine; almost certainly if they are brought across they will not give pp, but the scoring changes we've done over the last months are also a concern.

**Comment by Shirou27** [comment]
I just hope they will migrate it. I'm playing osu! Lazer since 2021 on my Android phone and all my play history was in there. Also I cannot redownload the beatmaps that I played before anymore if they remove it completely.

**Reply by OwenCMYK** [reply]
I would recommend screenshotting your top plays just in case. So you can go back and play them again if you need to
